### DISCUSSION TITLE
2022-12-14, v1.0.0 (initial)

### DISCUSSION THREAD

**Discussion by sauravhathi** [sauravhathi/oas.lpu-tool#2] [Q&A] [answered]
**Full Changelog**: https://github.com/sauravhathi/oas.lpu-tool/commits/v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/sauravhathi/oas.lpu-tool/releases/tag/v1.0.0'>2022-12-14, v1.0.0 (initial)</a>.</em>

**Comment by raviroushan4098** [comment, marked as answer]
download answer is not working

**Reply by sauravhathi** [reply]
![image](https://user-images.githubusercontent.com/61316762/216647130-7983354e-ff44-4c57-9e60-53897f4a2187.png)

read the last line
